figures/settings.py

```diff
@@ -2,28 +2,54 @@
 This module provides default values for running Figures.
```

Comment:
@OmarIthawi 
Figures needs items in django settings updated. Can you think of an option where we not doing one of the following:

A) Go with what I have. What I have is in part a response to your requests that I don't modify edx-platform
B) Make edx-platform envs pull from Figures settings, so we're adding more custom logic in lms/ends

AND make this so that members of the community don't have to start doing more than a package include in their forks?

I feel like you're putting me between a rock and a hard place. You don't want me to include customizations in edx-platform, but AFAIK, the only option I've got is to do those updates in Figures when the LMS starts up.

Am I missing something here?

Reply:
Can you mention which design anti-pattern this is doing?